domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/casefield/CaseFieldEntityACLValidatorImpl.java

```diff
@@ -14,12 +21,15 @@ public ValidationResult validate(final CaseFieldEntity caseField,
                                      final CaseFieldEntityValidationContext caseFieldEntityValidationContext) {
 
         final ValidationResult validationResult = new ValidationResult();
-
+        Set<String> caseRoles = caseFieldEntityValidationContext.getCaseRoles().stream().map(CaseRoleEntity::getReference).collect(Collectors.toSet());
         for (CaseFieldACLEntity entity : caseField.getCaseFieldACLEntities()) {
 
             if (null == entity.getUserRole()) {
                 validationResult.addError(new CaseFieldEntityInvalidUserRoleValidationError(entity,
                     new AuthorisationCaseFieldValidationContext(caseField, caseFieldEntityValidationContext)));
+            } else if (isCaseRole(entity.getUserRole().getReference()) && !caseRoles.contains(entity.getUserRole().getReference())) {
```

Comment:
does this mean that [CREATOR] needs to be declared as a case role in the CaseRoles spreadsheet tab now?

Reply:
Yes, that is correct.